src/Dto/Cell.php

```diff
@@ -30,6 +30,17 @@ public function addAttribute(Attribute $attribute): void
         $this->attributes[] = $attribute;
     }
 
+    public function setAttribute(Attribute $attribute): void
+    {
+        $this->attributes = [];
+        $this->attributes[] = $attribute;
+    }
+
+    public function getAttribute():array
+    {
+        return $this->attributes;
+    }
+
```

Comment:
I don't really like setting attributes, and overall this method is non-sense.
You either "set" -> full change of array, or "add", never both in the same method.